webui/react/src/hooks/useModal/Model/useModalModelCreate.tsx

```diff
@@ -171,9 +218,43 @@ const useModalModelCreate = ({ onClose }: Props = {}): ModalHooks => {
             <Form.Item
               label="Model name"
               name="modelName"
-              rules={[{ message: 'Model name is required ', required: true }]}>
+              required
+              requiredMessage="Model name is required ">
               <Input onChange={handleNameChange} />
             </Form.Item>
+            <Tooltip placement="bottom">
+              <Form.Item label="Workspaces" name="workspace">
+                <Dropdown
+                  arrow
+                  className={css.workspacDropdown}
+                  disabled={!workspaces.length || isWorkspace}
+                  dropdownRender={(menu) =>
+                    React.cloneElement(menu as React.ReactElement, {
+                      style: { maxHeight: '200px', maxWidth: 'fit-content', overflowY: 'scroll' },
```

Comment:
layout doesnt look right.
iirc, antd Select component has a good props. 
also, we dont support type to search? i think that would be nice to ask design team

<img width="624" alt="image" src="https://user-images.githubusercontent.com/109113616/218908110-5536ac97-455a-4829-9a9d-93872df82088.png">